vllm/v1/metrics/prometheus.py

```diff
@@ -56,8 +56,11 @@ def unregister_vllm_metrics():
     
     This is useful for testing and CI/CD where metrics may be registered
     multiple times across test runs.
+    
+    Also, in case of multiprocess, we need to unregister the metrics from the 
+    global registry.
     """
-    registry = get_prometheus_registry()
+    registry = REGISTRY
```

Comment:
It turns out that in multiprocess this still needs to be REGISTRY. The prev code complained with the following error: 
```
ValueError: Duplicated timeseries in CollectorRegistry: {'vllm:num_requests_running'}
```